src/poker.js

```diff
@@ -1,13 +1,82 @@
-function winningPair() {}
+const cardMapping  = {
+  J: '11',
+  Q: '12',
+  K: '13',
+  A: '14'
+}
+
+function winningPair(card1, card2) {
+  const card1Number = mapCardsToValues(card1)
+  const card2Number = mapCardsToValues(card2)
+
+  if (!hasPair(card1Number) && !hasPair(card2Number)) 
+    return []
+
+  if (hasPair(card1Number) && hasPair(card2Number)) {
+    return card1Number[0] > card2Number[0] ? card1 : card2
+  } else if (hasPair(card1Number)) {
+    return card1
+  } else {
+    return card2
+  }
+}
+
+function winningPairFromArray(array) {
+  const doublesArr = locateDoubles(array)
+
+  if (doublesArr.length === 0) 
```

Comment:
JS has truthy and falsy values. 0, empty strings, null, undefined are all considered falsy, so you could simplify this to

`if (!doublesArr.length)`